pkgs/development/python-modules/triton/binary-hashes.nix

```diff
@@ -0,0 +1,37 @@
+# Warning: Need to update at the same time as torch-bin
+#
+# Precompiled wheels can be found at:
+# https://download.pytorch.org/whl/torch_stable.html
+
+# To add a new version, run "prefetch.sh 'new-version'" to paste the generated file as follows.
+
+version:
+builtins.getAttr version {
+  "3.0.0" = {
+    x86_64-linux-38 = {
+      name = "triton-3.0.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl";
+      url = "https://files.pythonhosted.org/packages/ae/b2/048c9ecfdba0e6b0ae3c02eed2d9dd3e9e990a6d46da98555cf0c2232168/triton-3.0.0-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl";
```

Comment:
The old URLs were consuming from https://download.pytorch.org/whl/, can we keep doing that?

Reply:
we cant

Reply:
They do have some versions marked `cp312`, presumably that should be enough

Reply:
Sorry I'm being annoying, but can we do the bin hashes in a separate PR? It's such an obvious place to sneak in malware I wish we'd just ditched the whole thing...

TODO for myself:

- [ ] Confirm that installing from pytorch channels resolves into the pypi triton nowadayas